src/identity/apis/auth.ts

```diff
@@ -313,3 +315,29 @@ export const getDefaultAccountDefaultOrg = async (): Promise<
     throw new GenericError('No default account found')
   }
 }
+
+export const getUserAccounts = async (): Promise<UserAccount[]> => {
+  const response = await getAccounts({})
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+  if (!Array.isArray(response.data)) {
+    throw new GenericError('No account found')
+  }
+
+  return response.data
+}
+
+export const updateUserAccount = async (accountId, name) => {
+  const resp = await patchAccount({accountId, data: {name}})
+
+  if (resp.status !== 200) {
+    throw new Error(`Account rename update failed`)
+  }
```

Comment:
this can throw `401`, `422` and `500`, so let's handle those errors similar to how the other functions in this file handle errors

Reply:
updated